public/script.js

```diff
@@ -78,4 +55,22 @@ function validateForm() {
   return true;
 }
 
-document.getElementsByClassName('btnForm')[0].addEventListener('submit', validateForm);
\ No newline at end of file
+document.getElementsByClassName('btnForm')[0].addEventListener('submit', validateForm);
+
+// Activity 4 : Local storage
+const namef = document.getElementById('name');
+const email = document.getElementById('email');
+const message = document.getElementById('message');
+function storeValues() {
+  localStorage.setItem('f_name', namef.value);
+  localStorage.setItem('email', email.value);
+  localStorage.setItem('message', message.value);
+}
```

Comment:
- [ ] Kindly create a single JavaScript object with all the data from the entire form and save it in local storage. Do not create one local storage entry per input field.